src/tools/miri/src/shims/foreign_items.rs

```diff
@@ -928,6 +928,28 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.write_scalar(Scalar::from_f64(res), dest)?;
             }
 
+            "llvm.prefetch" => {
+                let [_, rw, loc, ty] =
```

Comment:
It is never okay to throw away an argument with `_`. At the least some `to_<int type>` method should be called on it to ensure it has the right size.